io/disk/softwareraid.py.data/Readme

```diff
@@ -10,14 +10,7 @@ Values to be passed in yaml file:
 
 * metadata levels for raid
 
-* Setup - True if only raid needs to be created and exit
-
-* Cleanup - True if only raid needs to be deleted and exit
-
-* Run_tests - True if only mdadm tests (like add, fail, remove device, etc)
-needs to be run on created raid
-
-* force - True if force option needs to be passed while creating raid.
+* spare disks [optional parameter]
 
 Note: Please specify minimum of 5 disks, for creating all levels of software
```

Comment:
Add here: the disk are separated by space. having the same size partitions is preferred.